web/README.md

```diff
@@ -46,6 +46,13 @@ rust using the above instructions).
 
 For the compiler to be able to output WebAssembly, an additional target has to be added to it: `rustup target add wasm32-unknown-unknown`
 
+#### Java
+
+Follow the instructions [to install OpenJDK](https://openjdk.org/projects/jdk/19/) on your machine.
```

Comment:
we only require headless JRE (not full JDK), right? Can mention that too.
(for example, on my Ubuntu, `openjdk-8-jre-headless` is enough)